app.py

```
@@ -58,36 +60,88 @@ async def login(user: UserLogin, db: Session = Depends(get_db)):
-    return {"success": True, "message": "로그인 성공!"}
+    return {"success": True, "message": "로그인 성공!", "user_id": db_user.user_id}
+
+# 기록 생성
+class TempDiaryData(BaseModel):
+    user_id: int
+    title: str
+    content: str
+    created_at: datetime
+
+@app.post("/api/temp_diary/create")
+async def create_temp_diary(data: TempDiaryData, db: Session = Depends(get_db)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)